src/JE.ApiValidation.WebApi.FluentValidation/JE.ApiValidation.WebApi.FluentValidation.nuspec

```diff
@@ -13,5 +13,11 @@
     <releaseNotes>https://github.com/justeat/JE.ApiValidation/blob/master/CHANGELOG.md</releaseNotes>
     <copyright>Copyright 2015</copyright>
     <tags>je-apivalidation api validation webapi</tags>
+    <dependencies>
```

Comment:
I didn't think it was necessary to include the dependencies element, since I thought `nuget pack` figured it out from the `packages.config` contents. If you check [public nuget.org for published versions](https://www.nuget.org/packages/JE.ApiValidation.WebApi/), dependencies are mentioned there, and I didn't do anything to set them up.